tests/test_snmp.py

```diff
@@ -235,6 +283,35 @@ def test_invalid_metric(self):
             self.assertEquals(sc['status'], self.check.CRITICAL, sc)
             self.assertEquals(sc['tags'], ['snmp_device:localhost'], sc)
 
+    def test_fallback_snmpgetnext(self):
```

Comment:
Good test!
Should we also test that `1.3.6.1.2.1.25.6.3.1.4` and `1.3.6.1.2.1.25.6.3.1.4.0` give the same value?